presto-spi/src/main/java/com/facebook/presto/spi/statistics/PlanStatistics.java

```diff
@@ -26,12 +26,14 @@
 @ThriftStruct
 public class PlanStatistics
 {
-    private static final PlanStatistics EMPTY = new PlanStatistics(Estimate.unknown(), Estimate.unknown(), 0);
+    private static final PlanStatistics EMPTY = new PlanStatistics(Estimate.unknown(), Estimate.unknown(), 0, Estimate.unknown(), Estimate.unknown());
 
     private final Estimate rowCount;
     private final Estimate outputSize;
     // A number ranging between 0 and 1, reflecting our confidence in the statistics
     private final double confidence;
+    private final Estimate hashTableNullCount;
```

Comment:
Call it something like `hashBuilderNullCount`. hashTable is too generic of a name.

Add comments here as well.